src/main/java/edu/ucsb/cs156/courses/controllers/CourseOverTimeInstructorController.java

```diff
@@ -0,0 +1,47 @@
+package edu.ucsb.cs156.courses.controllers;
+
+import java.util.List;
+
+import edu.ucsb.cs156.courses.collections.ConvertedSectionCollection;
+import edu.ucsb.cs156.courses.documents.ConvertedSection;
+import io.swagger.annotations.ApiOperation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@RestController
+@RequestMapping("/api/public/courseovertime")
+public class CourseOverTimeInstructorController {
+
+    private final Logger logger = LoggerFactory.getLogger(CourseOverTimeInstructorController.class);
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Autowired
+    ConvertedSectionCollection convertedSectionCollection;
+
+    @ApiOperation(value = "Get a list of courses over time, filtered by instructor name")
+    @GetMapping(value = "/instructorsearch", produces = "application/json")
+    public ResponseEntity<String> search(
+        @RequestParam String startQtr,
+        @RequestParam String endQtr,
+        @RequestParam String instructor
```

Comment:
See my feedback on PR #19 here: 
https://github.com/ucsb-cs156-s23/proj-courses-s23-7pm-3/pull/19#discussion_r1208653470

I think adding similar `@ApiParam` annotations will make this easier for future programmers to work with.